### ISSUE TITLE
/app/functions.sh: line 69: self_cid: unbound variable

### ISSUE THREAD

**Issue by graffen** [JrCs/docker-letsencrypt-nginx-proxy-companion#542]
I can't seem to be able to start the container anymore. I get the following in the log when I try: 
```
/app/functions.sh: line 69: self_cid: unbound variable
Error: can't get nginx-proxy container ID !
Check that you are doing one of the following :
        - Use the --volumes-from option to mount volumes from the nginx-proxy container.
        - Set the NGINX_PROXY_CONTAINER env var on the letsencrypt-companion container to the name of the nginx-proxy container.
        - Label the nginx-proxy container to use with 'com.github.jrcs.letsencrypt_nginx_proxy_companion.nginx_proxy'.
```

My docker commandline is as follows: 
```
docker run -d --name nginx-proxy-letsencrypt --volumes-from nginx-proxy -v /var/run/docker.sock:/var/run/docker.sock:ro jrcs/letsencrypt-nginx-proxy-companion
```

I've been trying tonnes of permutations of this over the last few hours and now I'm ready to give up. Does anyone have an idea what might be going on? Let me know if you require any additional info.

**Comment by buchdag** [comment]
What version of the container are you using ?

**Comment by graffen** [comment]
:latest (sorry, forgot to note that I just updated my entire stack)

**Comment by buchdag** [comment]
What version where you using prior to the update ?

Could you try with `v1.11` and `v1.10` ?

**Comment by graffen** [comment]
Ah! `v1.10` seems to work!

**Comment by buchdag** [comment]
I think I identified the issue, can you try with `duch/letsencrypt-nginx-proxy-companion:542` ?

**Comment by graffen** [comment]
That looks like it works, yes.

**Comment by buchdag** [comment]
Fix committed to master / `latest`

Released `v1.11.1` with the fix and pushed it to Dockerhub as both `v1.11` and `v1.11.1`.

Thanks for reporting the issue.

**Comment by buchdag** [comment]
Try to stick to tagged release whenever possible, using `latest` tends to make issue identification harder.
